src/ChluIPFS.js

```diff
@@ -69,29 +73,47 @@ class ChluIPFS {
             await this.persistence.persistData();
         }
 
-        // If customer, also wait for at least one peer to join the room (TODO: review this)
         if (this.type === constants.types.customer) {
-            await this.room.waitForAnyPeer();
             // Broadcast my review updates DB, but don't fail if nobody replicates
             this.room.broadcastReviewUpdates(false);
         } else if (this.type === constants.types.service) {
             await this.serviceNode.start();
         }
-
+        this.ready = true;
+        this.starting = false;
+        this.events.emit('ready');
         return true;
     }
 
     async stop() {
+        this.events.emit('stopping');
+        this.ready = false;
         await this.serviceNode.stop();
```

Comment:
@fazo96 should we be handling errors responses from any of these stop calls? Are we sure they _never_ result in an error?

Reply:
If we are sure that chlu-demo will handle any potential error from here and show a meaningful message to the user or try to recover from error in some other way, then I am fine letting the low level errors bubble up to the application.

Otherwise, we could consider having an error type that chlu-demo understands and can show a toast message to the user.

We can leave this for now, as the only place stop is called during the demo is when we switch the node type.